lamindb/models/writelog.py

```diff
@@ -52,8 +52,7 @@ class WriteLog(models.Model):
     created_by_uid = models.CharField(max_length=8, default=DEFAULT_CREATED_BY_UID)
     branch_code = models.IntegerField(default=DEFAULT_BRANCH_CODE)
     run_uid = models.CharField(max_length=16, default=DEFAULT_RUN_UID)
-    # Many-to-many tables don't have row UIDs, so this needs to be nullable.
-    record_uid = models.JSONField(null=True)
+    record_uid = models.JSONField()
```

Comment:
I'm only seeing this now, but I think this should likely be a `CharField` rather than a `JSONField`, should it?

Reply:
I tried this in the next PR and it seems there is a good reason for that this is a JSON field. Many tests break if turning it into CHARField. I think this is because you're modeling multiple uids as a list within the JSON field.

https://github.com/laminlabs/lamindb/pull/2798

Reply:
The write log record's UID takes one of two forms:

* For tables that are many to many, it's a list of 3-tuples specifying the table's foreign-key relationships by UID
* For all other tables, it's an ordered list of values of the table's UID fields.

Because of that it's easier if those fields are JSONFields rather than CharFields.